Makefile

```diff
@@ -13,6 +13,7 @@ deps:
 	docker-compose run --rm cli composer install
 
 depsfortests:
+	docker-compose run --rm appfortests composer update
```

Comment:
I'm a little hesitant to ever automatically update dependencies, though it's also very tempting and could be a good choice. Was there a particular motivator for adding this here?